INSTALL.md

```diff
@@ -164,6 +164,18 @@ Then you can try one of the
 [examples](https://github.com/kubernetes/client-go/tree/v2.0.0/examples/) from
 the 2.0.0 release.
 
+For client-go 4.0.0 must be defined `k8s.io/apimachinery` constraint in Gopkg.toml
```

Comment:
For client-go 4.0.0, a `k8s.io/apimachinery` constraint must be defined in Gopkg.toml

Reply:
Also could you squash the commits? Then lgtm. 

Sorry for the delay in the response.